core/src/com/cloud/agent/api/StartupRoutingCommand.java

```diff
@@ -35,7 +35,7 @@
     long memory;
     long dom0MinMemory;
     boolean poolSync;
-
+    private boolean _supportsClonedVolumes;
```

Comment:
+1 as well.  Ridding of odd Hungarian notation and "_" would be a very things for the codebase.